src/it/scala/org/zalando/spearheads/innkeeper/dao/RoutesPostgresRepoSpec.scala

```diff
@@ -6,10 +6,10 @@ import akka.actor.ActorSystem
 import akka.stream.ActorMaterializer
 import akka.stream.scaladsl.Source
 import org.scalatest.concurrent.ScalaFutures
-import org.scalatest.time.{ Seconds, Span }
-import org.scalatest.{ BeforeAndAfter, FunSpec, Matchers }
```

Comment:
why is there different formatting? probably a mismatch between scalastyle (applied at compile time) and IDEA formatting. Good idea to unite those to avoid such changes